janusgraph-test/src/main/java/org/janusgraph/diskstorage/indexing/IndexProviderTest.java

```diff
@@ -966,7 +965,7 @@ public void testCustomAnalyzer() throws Exception {
             query = new IndexQuery(store, PredicateCondition.of(FULL_TEXT, Text.CONTAINS, "Tom Jerry"));
             assertEquals(query.toString(), 1, tx.query(query).size());
             query = new IndexQuery(store, PredicateCondition.of(FULL_TEXT, Text.CONTAINS, "Tom or Jerry"));
-            assertEquals(query.toString(), 1, tx.query(query).size());
+            assertEquals(query.toString(), 0, tx.query(query).size());
```

Comment:
Why is this now `0`?

Reply:
Because I change the analyzer of TEXT from English to default analyzer, so 'or' is no longer a stop word.
I change this to be sure the analyzer of STRING and TEXT are different. So that tests which field (normal or __STRING) are query by equals and Contains.